emc-metalnx-core/src/main/java/com/emc/metalnx/core/domain/entity/enums/DataGridPermType.java

```diff
@@ -9,8 +9,17 @@
  * */
 
 public enum DataGridPermType {
-
-    OWN("OWN"), WRITE("WRITE"), READ("READ"), IRODS_ADMIN("ADMIN"), NONE("NONE");
+	
+	NONE("NONE"),
+	READ_METADATA("READ_METADATA"),
+	READ_OBJECT("READ_OBJECT"),
```

Comment:
Hmm, depending on how this is used, we probably have to maintain READ/WRITE in addition to these for backward compatibility.

How does Metalnx use these?
Do they make it to the browser in any way?

Reply:
They do.  I will have to test this with iRODS 4.2.x.  I believe Terrell said it was ok to release this and patch it if it doesn't work properly with 4.2.x.

Reply:
True. Let's test it with 4.2.12 and see how we feel after that.